src/components/Menu/Menu/Menu.tsx

```diff
@@ -50,7 +50,7 @@ const Menu: VibeComponent<MenuProps> & {
 } = forwardRef(
   (
     {
-      id,
+      id = "menu",
```

Comment:
It's not standard to set default id to a react component. It can cause a case of multiple elements with the same id in the page, which can cause a11y issues and JS issues because id should be unique per element

Reply:
I understand that we need the menu key for accessibility, maybe we can use here third packages for generate id dynamically as uuid? I'm not sure if it's an overkill, let's talk about it in the daily

Reply:
dynamic uuid added
haven't implemented warnings - I think it's too noisy and not really required here